File: f/src/types/ShopOrder.ts

type ShopOrder = {
  transaction_id?: number
  cart_id?: number,
  quantity?: number,
  product_id?: number,
  product_name?: string,
  product_image?: string
  product_price?: number,
  user_id?: number,
  delivery_status?: string
}

export default ShopOrder

File: b/model/ShopOrder.go
package model

import "gorm.io/gorm"

type ShopOrder struct {
	gorm.Model
	TransactionId  uint   `json:"transaction_id"`
	CartID         uint   `json:"cart_id"`
	Quantity       uint   `json:"quantity"`
	ProductId      uint   `json:"product_id"`
	ProductName    string `json:"product_name"`
	ProductImage   string `json:"product_image"`
	ProductPrice   uint   `json:"product_price"`
	UserId         uint   `json:"user_id"`
	DeliveryStatus string `json:"delivery_status"`
}
